bliss/cached_dataset.py

```diff
@@ -149,7 +149,10 @@ def __getitem__(self, index):
             self.buffered_file_index = converted_index
             with open(self.file_paths[converted_index], "rb") as f:
                 self.buffered_data = torch.load(f)
-        output_data = self.buffered_data[converted_sub_index]
+        try:
+            output_data = self.buffered_data[converted_sub_index]
+        except KeyError:
```

Comment:
Why are we getting key errors here?

Reply:
investigating. was initially done as a workaround for file names, perhaps redundant now. will fix if so

Reply:
waiting for data gen to fix this, still outstanding -- will test & fix first thing tomorrow